packages/bits/yarn.lock

```diff
@@ -523,7 +523,7 @@
     postcss-value-parser "4.1.0"
     throat "5.0.0"
 
-"@axe-core/webdriverjs@4.1.1":
+"@axe-core/webdriverjs@^4.1.1":
```

Comment:
Please revert back yarn.lock

Reply:
discussed offline. circle ci made me do this in order to have a successful build